### DISCUSSION TITLE
Lock mouse cursor into window/lock remote cursor to the actual mouse cursor (Like in games, etc.)

### DISCUSSION THREAD

**Discussion by discoveryX64** [rustdesk/rustdesk#5909] [Feature Request]
Title. Don't know how to explain it further, I guess something like how steamlink locks your cursor to its window?

**Comment by CatMan6112** [comment]
its been 2 years 😭

**Comment by Vuthric** [comment]
Baffling that this is not decently prioritized. We realized rustdesk is "not designed for cloud gaming". Few of us have that in mind but a lot of us do work in gamedev. Testing out bugs in my coworkers' machines has become a real struggle. No controller support or mouse lock. I hate to admit it but maybe Anydesk is better after all.
